portal/www/portal/wireless_operations.php

```diff
@@ -0,0 +1,164 @@
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and/or hardware specification (the "Work") to
+// deal in the Work without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Work, and to permit persons to whom the Work
+// is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Work.
+//
+// THE WORK IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+// OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+// WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE WORK OR THE USE OR OTHER DEALINGS
+// IN THE WORK.
+//----------------------------------------------------------------------
+?>
+<?php
+
+// Support for wireless operations for enabling/disabling projects
+//   in ORBIT/Wireless/WiMAX testbeds and 
+//   synchronizing project/member information between GENI and ORBIT
+// 
+//  wireless_operations?operation=sync&project_name=X
+//    synchronize project data for project X between GENI and ORBIT
+//
+//  wireless_operations?operation=enable&project_name=X&project_id=XID
+//    enable project X for wireless operations
+//
+//  wireless_operations?operation=disable&project_name=X&project_id=XID
+//    disable project X for wireless operations
+// 
+
+?>
+
+<?php
+
+
+require_once('user.php');
+require_once('sr_constants.php');
+require_once('sr_client.php');
+require_once("pa_client.php");
+require_once("pa_constants.php");
+require_once("response_format.php");
+
+$user = geni_loadUser();
+if (!isset($user) || is_null($user) || ! $user->isActive()) {
+  relative_redirect('home.php');
+}
+$ma_url = get_first_service_of_type(SR_SERVICE_TYPE::MEMBER_AUTHORITY);
+$sa_url = get_first_service_of_type(SR_SERVICE_TYPE::SLICE_AUTHORITY);
+
+
+// Invoke geni-sync-wireless tool on given project
+function sync_object($object_type, $object_name)
+{
+  $cmd = "geni-sync-wireless $object_type $object_name";
+  //  error_log("SYNC(cmd) " . $cmd);
+  $descriptors = array(0 => array("pipe", "r",), 
+		 1 => array("pipe", "w"), 
+		 2 => array("pipe", "w"));
+  $process = proc_open($cmd, $descriptors, $pipes);
+  $out = stream_get_contents($pipes[1]);
+  $err = stream_get_contents($pipes[2]);
+  fclose($pipes[1]);
+  fclose($pipes[2]);
+  $proc_value = proc_close($process);
+  $full_output = $out . $err;
+
+  //  error_log("OUT = " . $out);
+  //  error_log("ERR = " . $err);
+  //  error_log("PROC_VALUE = " . $proc_value);
+  
+  foreach(split("\n", $full_output) as $line) {
+    error_log("SYNC(output) " . $line);
```

Comment:
Do we really want to log everything when we do a wireless sync? That seems verbose.